README.md

```diff
@@ -1,69 +1,103 @@
-# Theia applications
+<!-- Main Header  -->
+<div align='center'>
+
+<img src="./assets/theia.svg" width="125px">
+
+## Theia Applications - Docker & Electron
+
 [![Build Status](https://travis-ci.org/theia-ide/theia-apps.svg?branch=master)](https://travis-ci.org/theia-ide/theia-apps)
 
-[Theia](https://github.com/theia-ide/theia) is a platform to develop Cloud & Desktop IDEs with modern web technologies.
+Collection of Theia-based example applications based on published Theia extension packages.
+
+</div>
+
+---
 
-This repository contains example Theia applications based on published Theia extension packages.
+<br />
 
-- [Theia Docker](#theia-docker)
-  - [How to use `theiaide/theia` image?](#how-to-use-theiaidetheia-image)
-  - [Image Variants](#image-variants)
-    - [`theiaide/theia:latest`](#theiaidetheialatest)
-    - [`theiaide/theia:<version>`](#theiaidetheiaversion)
-    - [`theiaide/theia:next`](#theiaidetheianext)
-    - [`theiaide/theia:<version>-next.<commit hash>`](#theiaidetheiaversion-nextcommit-hash)
-- [Theia Desktop](#theia-desktop)
-- [Contributing](CONTRIBUTING.md)
+### Outline
+
+- [Docker Image Variants](#docker-image-variants)
+- [Electron Applications](#electron-apps)
+- [Building & Running](#building-&-running)
+- [Tips & Tricks](#tips-&-tricks)
+- [Contributing](#contributing)
 - [License](#license)
 
-## Theia Docker
+<br />
+
+### Docker Image Variants
+
+<img
+    src='./assets/docker.png'
+    alt='docker logo'
+    width="125px"
+/>
+
+| Image Name | Description | Documentation |
+|:---|:---|:---|
+| [theiaide/theia-cpp](https://hub.docker.com/r/theiaide/theia-cpp) | Theia-based C/C++ example application | [docs](./theia-cpp-docker/README.md) |
+| [theiaide/theia-dart](https://hub.docker.com/r/theiaide/theia-dart) | Theia-based Dart example application | |
+| [theiaide/theia-full](https://hub.docker.com/r/theiaide/theia-full) | Theia-based example application with all available published applications | |
+| [theiaide/theia-go](https://hub.docker.com/r/theiaide/theia-go) | Theia-based Go example application | [docs](./theia-go-docker/README.md) |
+| [theiaide/theia-java](https://hub.docker.com/r/theiaide/theia-java) | Theia-based Java example application | |
+| [theiaide/theia-python](https://hub.docker.com/r/theiaide/theia-python) | Theia-based Python example application | [docs](./theia-python-docker/README.md) |
+| [theiaide/theia-ruby](https://hub.docker.com/r/theiaide/theia-ruby) | Theia-based Ruby example application | |
+| [theiaide/theia-rust](https://hub.docker.com/r/theiaide/theia-rust) | Theia-based Rust example application | [docs](./theia-rust-docker/README.md) |
+| [theiaide/theia-swift](https://hub.docker.com/r/theiaide/theia-swift) | Theia-based Swift example application | |
+| [theiaide/theia](https://hub.docker.com/r/theiaide/theia) | Theia-based JavaScript/TypeScript (Web Technologies) example application | |
 
```

Comment:
What about the other docker images found in the same folder:
- theia-deb-build-docker
- theia-https-docker
- theia-openshift-docker
- theia-rpm-build-docker
- yangster-docker
 What should we do about hem !

Reply:
A lot of the images are simply examples on how to do specific things (they are not actually published):


> * theia-deb-build-docker

Demonstrates how to build `debian` packages.

> * theia-https-docker

Demonstrates how to add security to existing images.

> * theia-openshift-docker

Possible deprecated (not much content)

> * theia-rpm-build-docker

Demonstrates how to build `rpm` packages.

@lmcbout  perhaps I should have an `other` section of examples?

Reply:
Nice suggestion, another group for "Examples"

Reply:
I've updated this section if you'd like to check.